src/com/amazon/ion/benchmark/ReadGeneralConstraints.java

```diff
@@ -7,70 +7,73 @@
 import com.amazon.ion.IonStruct;
 import com.amazon.ion.IonSystem;
 import com.amazon.ion.IonType;
+import com.amazon.ion.IonValue;
 import com.amazon.ion.system.IonReaderBuilder;
 import com.amazon.ion.system.IonSystemBuilder;
 
 import java.io.BufferedInputStream;
 import java.io.FileInputStream;
 
 /**
- * Parse Ion schema file and get the general constraints in the file then pass the constraints to the Ion data generator.
+ * Parse Ion Schema file and get the general constraints in the file then pass the constraints to the Ion data generator.
  */
 public class ReadGeneralConstraints {
+    private static final IonSystem system = IonSystemBuilder.standard().build();
+    private static final IonLoader loader = system.newLoader();
+    private static final String defaultRange = "[0, 1114111]";
```

Comment:
Minor: Java style recommends all-caps snake case for constants, e.g. `SYSTEM`, `DEFAULT_RANGE`.

Reply:
> Minor: Java style recommends all-caps snake case for constants, e.g. `SYSTEM`, `DEFAULT_RANGE`.

Got it . I'll update these changes in the next PR